### PR TITLE
Added transitions to cancellation form

### PR DESCRIPTION
**Pull request by piekczyk**
# [Added transitions to cancellation form](https://app.shortcut.com/oazo-apps/story/3637/cancellation-of-downside-protection-completed)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- Added transitions to cancellation form
  
## How to test 🧪
  <Please explain how to test your changes>
- heroku / locally
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -810,11 +807,19 @@
   "protection": {
     "navigate-cancel": "Cancel downside protection",
     "navigate-adjust": "Return to adjusting stop loss",
+    "set-downside-protection": "Set downside protection",
```
Comment (johnnieskywalker) [inline public/locales/en/common.json:810]: [minor] Please add the same keys to es, pt, cn commons

**Review by johnnieskywalker — CHANGES_REQUESTED** [review]

File: features/automation/controls/CancelSlFormControl.tsx [hunk 2]
```diff
@@ -121,16 +124,23 @@ export function CancelSlFormControl({
     isStopLossEnabled,
   }
 
+  const token = vault.token
```
Comment (rkrasiuk) [inline features/automation/controls/CancelSlFormControl.tsx:127]:
```suggestion
  const { token } = vault
```

File: features/automation/controls/CancelSlFormLayout.tsx [hunk 3]
```diff
@@ -1,57 +1,147 @@
...
+  liquidationPrice,
+}: CancelDownsideProtectionInformationProps) {
+  const { t } = useTranslation()
+
+  return (
+    <VaultChangesInformationContainer title={t('cancel-stoploss.summary-header')}>
+      <VaultChangesInformationItem
+        label={`${t('cancel-stoploss.liquidation')}`}
+        value={<Flex>${formatAmount(liquidationPrice, 'USD')}</Flex>}
+      />{' '}
```
Comment (rkrasiuk) [inline features/automation/controls/CancelSlFormLayout.tsx:40]: is this intentional?

File: public/locales/en/common.json (hunk 1 above)
Comment (piekczyk) [reply public/locales/en/common.json:810]: as long as keys are present only in EN, translations will fallback to it so it doesn't make sense to add it everywhere

File: public/locales/en/common.json [hunk 4]
```diff
@@ -810,11 +807,19 @@
...
+    "set-downside-protection-desc": "Some intro text. Some more text, even more, even more, even more, even more, even more, even more, even more {{ link }}",
+    "cancel-downside-protection": "Cancel downside protection",
+    "cancel-downside-protection-desc": "To cancel the stop loss you’ll need to click the button below and confirm the transaction.",
     "setting-downside-protection": "Setting downside protections",
+    "canceling-downside-protection": "Canceling stop loss protections",
     "setting-downside-protection-desc": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras iaculis quam quis risus finibus, non imperdiet.",
+    "canceling-downside-protection-desc": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras iaculis quam quis risus finibus, non imperdiet.",
```
Comment (rkrasiuk) [inline public/locales/en/common.json:817]: nit
```suggestion
    "cancelling-downside-protection": "Canceling stop loss protections",
    "setting-downside-protection-desc": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras iaculis quam quis risus finibus, non imperdiet.",
    "cancelling-downside-protection-desc": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras iaculis quam quis risus finibus, non imperdiet.",
```

**Comment by johnnieskywalker** [comment]
image

**Review by johnnieskywalker — APPROVED** [review]
image

**Pull request merged**